### PR TITLE
Fixed container base image

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Review by gburges — APPROVED** [review]

File: nginx.conf [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-# ~ ^\/apps\/([^\/]+)
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    server {
```
Comment (Victoremepunto) [inline nginx.conf:15]:
```suggestion
    server {
```

File: nginx.conf [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
 }
 
-# ~ ^\/apps\/([^\/]+)
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
```
Comment (Victoremepunto) [inline nginx.conf:13]: mind to mention why are we adding these ?

File: nginx.conf [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
- access_log /dev/stdout;
-
- server_name localhost;
- ssl_certificate /var/serving-cert/tls.crt;
- ssl_certificate_key /var/serving-cert/tls.key;
-
- location / {
-  alias /opt/clowder-plugin/;
- }
+worker_processes  1;
```
[the remark this answers is not in the document]
Comment (Victoremepunto) [reply nginx.conf:1]:
```suggestion
```

File: nginx.conf [hunk 4]
```diff
@@ -1,15 +1,28 @@
...
+        server_name localhost;
+        ssl_certificate /var/serving-cert/tls.crt;
+        ssl_certificate_key /var/serving-cert/tls.key;
+
+
+        location / {
+            alias /opt/clowder-plugin/;
+        }
+    }
+}
```
Comment (psav) [inline nginx.conf:28]:
```suggestion
}

```

File: nginx.conf (hunk 2 above)
Comment (psav) [reply nginx.conf:12]: because we are taking away the defaults file - I thought it would be good to make it explicit

Comment (Victoremepunto) [reply nginx.conf:12]: Sorry, it isn't clear to me as these values differ from Nginx's default values. `keepalive_timeout` for example, is set to 65, but [the default value is 75](http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout). 
I just wanted to understand what motivated this fine tuning

**Pull request merged**